components/ucd/Cargo.toml

```diff
@@ -18,3 +18,4 @@ unic-ucd-bidi = { path = "bidi/", version = "0.4.0" }
 unic-ucd-core = { path = "core/", version = "0.4.0" }
 unic-ucd-normal = { path = "normal/", version = "0.4.0" }
 unic-ucd-utils = { path = "utils/", version = "0.4.0" }
+unic-ucd-category = { path = "category/", version = "0.0.1" }
```

Comment:
Right. I think you can start with `0.4.0`. There won't be a release of it, but it allows the code work and we will update it to `0.5.0` for the next general release.